libs/java/auth_core/pom.xml

```diff
@@ -105,4 +105,26 @@
     </dependency>
   </dependencies>
 
+  <profiles>
+    <profile>
+      <id>arm-os-build</id>
+      <activation>
+        <os>
+          <arch>aarch64</arch>
+        </os>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <version>${maven-surefire-plugin.version}</version>
+            <configuration>
+              <skipTests>true</skipTests>
```

Comment:
Is it possible to skip only UserAuthorityTest?
I know that it is possible to achieve it in code..

Reply:
Fixed it.